packages/neuron-ui/src/components/BroadcastTransaction/index.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useCallback, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useNavigate } from 'react-router-dom'
+import { isSuccessResponse, RoutePath, useDidMount, useGoBack } from 'utils'
+import Dialog from 'widgets/Dialog'
+import AlertDialog from 'widgets/AlertDialog'
+import { useDispatch, useState as useGlobalState } from 'states'
+import { broadcastTransaction, getCurrentWallet, OfflineSignStatus } from 'services/remote'
+import { ReactComponent as HardWalletIcon } from 'widgets/Icons/HardWallet.svg'
+
+import styles from './broadcastTransaction.module.scss'
+
+const BroadcastTransaction = () => {
+  const {
+    app: { loadedTransaction = {} },
+  } = useGlobalState()
+
+  const [wallet, setWallet] = useState<State.Wallet | null>(null)
+  const [isBroadCasting, setIsBroadcasting] = useState(false)
+  const [t] = useTranslation()
+  const dispatch = useDispatch()
+  const [errMsg, setErrMsg] = useState('')
+
+  const { filePath, json } = loadedTransaction
+
+  const jsonContent = useMemo(() => {
+    return JSON.stringify(json, null, 2)
+  }, [json])
+
+  const signStatus: OfflineSignStatus = json.status
+
+  const isSigned = useMemo(() => signStatus === OfflineSignStatus.Signed, [signStatus])
+
+  const onBack = useGoBack()
+
+  const navigate = useNavigate()
+  const onBroadcast = useCallback(async () => {
+    setIsBroadcasting(true)
+    try {
+      const res = await broadcastTransaction({
+        ...json,
+        walletID: wallet!.id,
+      })
+      if (isSuccessResponse(res)) {
+        navigate(RoutePath.History)
```

Comment:
https://github.com/nervosnetwork/neuron/pull/3000/files#r1435815632. Then I guess here we'd better not navigate to another page.

Reply:
Agree, as mentioned at https://github.com/nervosnetwork/neuron/pull/3000#discussion_r1435815632, the transaction to broadcast may not be related to any wallet in Neuron. So navigation to history could be misleading.

The best solution is as follows:
1. if the transaction has a cell of the current wallet, go to the history; Otherwise,
2. show a `View in Explorer` button to visit the transaction on CKB Explorer